db/schema.rb

```diff
@@ -608,14 +604,17 @@
   end
 
   create_table "course_survey_questions", force: :cascade do |t|
-    t.integer  "actable_id"
-    t.string   "actable_type", :limit=>255, :index=>{:name=>"index_course_survey_questions_actable", :with=>["actable_id"], :unique=>true}
-    t.integer  "survey_id",    :null=>false, :index=>{:name=>"fk__course_survey_questions_survey_id"}, :foreign_key=>{:references=>"course_surveys", :name=>"fk_course_survey_questions_survey_id", :on_update=>:no_action, :on_delete=>:no_action}
-    t.text     "description",  :null=>false
-    t.integer  "creator_id",   :null=>false, :index=>{:name=>"fk__course_survey_questions_creator_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_survey_questions_creator_id", :on_update=>:no_action, :on_delete=>:no_action}
-    t.integer  "updater_id",   :null=>false, :index=>{:name=>"fk__course_survey_questions_updater_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_survey_questions_updater_id", :on_update=>:no_action, :on_delete=>:no_action}
-    t.datetime "created_at",   :null=>false
-    t.datetime "updated_at",   :null=>false
+    t.integer  "survey_id",     :null=>false, :index=>{:name=>"fk__course_survey_questions_survey_id"}, :foreign_key=>{:references=>"course_surveys", :name=>"fk_course_survey_questions_survey_id", :on_update=>:no_action, :on_delete=>:no_action}
+    t.text     "description",   :null=>false
+    t.integer  "creator_id",    :null=>false, :index=>{:name=>"fk__course_survey_questions_creator_id"}, :foreign_key=>{:references=>"users", :name=>"fk_course_survey_questions_creator_id", :on_update=>:no_action, :on_delete=>:no_action}
```

Comment:
Typically the userstamps are kept as the last few fields, maybe we can stick to that convention? I'd think it would make reading tables in `schema.rb` slightly easier.